### PR TITLE
Languagepage

### PR DESCRIPTION
**Pull request by Sujith-Barve**
Added Header of the Language Select Page Header and Button name is added in only 5 languages

### CODE REVIEW

File: locale/English.json [hunk 1]
```diff
@@ -64,7 +65,8 @@
       "aboutDhiway": "About Dhiway",
       "langChange": "Select your language",
       "defaultScreenPick": "Activity as default screen",
-      "recordActivity": "Record Activity"
+      "recordActivity": "Record Activity",
+      "buttonName" : "Continue"
```
Comment (amarts) [inline locale/English.json]: confusing key...  Move it to 'common', and key name can be 'continueLabel'

File: locale/English.json [hunk 2]
```diff
@@ -4,7 +4,8 @@
       "Scan": "Scan",
       "Settings": "Settings",
       "Activity": "Activity",
-      "seqrByDhiway": "SEQR by DHIWAY"
+      "seqrByDhiway": "SEQR by DHIWAY",
+      "LanguagePage" : "Choose Language"
```
Comment (amarts) [inline locale/English.json]: couldn't we use `langChange` in Line 66?

**Review by amarts — CHANGES_REQUESTED** [review]
No need to commit changes to All of languages... lets make only change in English.json, and all other changes to languages come from poeditor...

File: locale/English.json (hunk 2 above)
Comment (Sujith-Barve) [reply locale/English.json]: Can we use again langChange as a header??

Comment (amarts) [reply locale/English.json]: We can. Lets hear from others on what they think.

**Comment by amarts** [comment]
This too has conflict with 'main' branch.. can you rebase to 'origin/main' here?

**Pull request closed without merging**